README.md

```diff
@@ -1,6 +1,14 @@
 # AI Multimodal Public Demos
 
-The demos in this repostiory rely solely on public data sets.
+This repository is part of [AIMM project](https://github.com/AI-multimodal): a joint and collaborative effort between scientist from [Brookhaven National Laboratory (BNL)](https://www.bnl.gov/) and [Lawrence Berkeley National Laboratory (LBL)](https://www.lbl.gov/)
+Current contributors working in this project are: Dylan McReynolds and Jospeh Kleinhenz from LBL, and Dan Allan and Juan Marulanda from BNL.
+Our goal is to create database for XAS data that can be used by scientist from beamlines at different Synchrtron Light Sources around the world. They will be able to use this database as a platform to share and use data to improve their research. 
+
+We develop and used [tiled](https://blueskyproject.io/tiled/) for these demos. First, we can define Tiled as a data access service for data science tools and we use it to put together all the different data sets that we receive from beamline scientists and it helps users to avoid worrying about formatting, file structure and parsing.
```

Comment:
I think we should say "integrate with" but not "develop" because core development of TIled is paid for by NSLS-II Operations, not AIMM.

Reply:
Agreed with @danielballan  We are building a funcitonal database of datasets from a variety of experiments and simulations. Oh, and we use Tiled.